src/main/resources/db/migration/20221124093822961__changed_template_parameter_names_to_lowercase.sql

```diff
@@ -0,0 +1,23 @@
+UPDATE reports.jasper_templates
```

Comment:
Should be reports.template_parameters

Reply:
Thanks for pointing this out, I'll fix this in a sec